addons/hw_drivers/tools/helpers.py

```diff
@@ -106,12 +100,28 @@ def check_certificate():
 
     try:
         with path.open('r') as f:
-            cert = crypto.load_certificate(crypto.FILETYPE_PEM, f.read())
+            return crypto.load_certificate(crypto.FILETYPE_PEM, f.read())
     except EnvironmentError:
         _logger.exception("Unable to read certificate file")
         return {"status": CertificateStatus.ERROR,
                 "error_code": "ERR_IOT_HTTPS_CHECK_CERT_READ_EXCEPTION"}
 
+
+def check_certificate():
+    """
+    Check if the current certificate is up to date or not authenticated
+    :return CheckCertificateStatus
+    """
```

Comment:
learnt from xmo comments, we should set a blank line between the description and the params/return :smile:  
(however, apparently, you don't need to need to separate params from return)
```suggestion
    """Check if the current certificate is up to date or not authenticated
    
    :return CheckCertificateStatus
    """
```